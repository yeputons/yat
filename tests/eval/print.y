def my_print(a) {
  print a + 10;
}
print my_print(100 + my_print(1));
