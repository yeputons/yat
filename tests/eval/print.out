11
121
121
